tests/test_click_events.py

```diff
@@ -0,0 +1,31 @@
+def setup_app(app):
+    from lona.html import Button
+    from lona import LonaView
+
+    @app.route('/')
+    class MyLonaView(LonaView):
+        def handle_request(self, request):
+            self.show(Button('click me!'))
+            self.await_click()
+            self.show('SUCCESS')
+
+    app.settings.SESSIONS = False
```

Comment:
nit: Why is it necessary?

Reply:
The Lona session middleware checks for a specific cookie to authenticate users. If the cookie is not set, the middleware sets it and redirects the browser, so the cookie gets active. The playwright browser seems not to follow this redirect. Maybe this has to be enabled, or the middleware has to be fixed.

Reply:
Test passes on my machine without this like.
According to [docs](https://playwright.dev/python/docs/api/class-page#page-goto), redirection should work well.
> Returns the main resource response. In case of multiple redirects, the navigation will resolve with the response of the last redirect.

Reply:
You are right, on mine too now. Then i did something wrong yesterday. I removed the line.